addons/stock/views/stock_picking_views.xml

```diff
@@ -380,13 +380,17 @@
-                                    <button type="object" name="action_product_forecast_report" icon="fa-area-chart text-danger" attrs="{'invisible': ['|', ('forecast_availability', '&gt;=', 0), '|', ('parent.immediate_transfer', '=', True), ('parent.picking_type_code', '=', 'outgoing')]}"/>
-                                    <field name="reserved_availability" string="Reserved" attrs="{'column_invisible': (['|','|', ('parent.state','=', 'done'), ('parent.picking_type_code', 'in', ['incoming', 'outgoing']), ('parent.immediate_transfer', '=', True)])}"/>
+                                    <button type="object" name="action_product_forecast_report" icon="fa-area-chart" 
+                                        attrs="{'invisible': ['|', ('forecast_availability', '&lt;', 0), '|', ('parent.immediate_transfer', '=', True), '&amp;', ('parent.picking_type_code', '=', 'outgoing'), ('state', '!=', 'draft')]}"/>
+                                    <button type="object" name="action_product_forecast_report" icon="fa-area-chart text-danger" 
+                                        attrs="{'invisible': ['|', ('forecast_availability', '&gt;=', 0), '|', ('parent.immediate_transfer', '=', True), '&amp;', ('parent.picking_type_code', '=', 'outgoing'), ('state', '!=', 'draft')]}"/>
+                                    <field name="forecast_expected_date" invisible="1"/>
+                                    <field name="forecast_availability" string="Reserved"
+                                        attrs="{'column_invisible': ['|', '|', ('parent.state', 'in', ['draft', 'done']), ('parent.picking_type_code', '!=', 'outgoing'), ('parent.immediate_transfer', '=', True)]}" widget="forecast_widget"/>
+                                    <field name="reserved_availability" string="Reserved"
+                                        attrs="{'column_invisible': ['|', '|', ('parent.state', 'in', ['draft', 'done']), ('parent.picking_type_code', 'in', ['incoming', 'outgoing']), ('parent.immediate_transfer', '=', True)]}"/>
                                     <field name="product_qty" invisible="1" readonly="1"/>
-                                    <field name="forecast_expected_date" invisible="1" />
-                                    <field name="forecast_availability" string="Reserved" attrs="{'column_invisible': ['|', '|', ('parent.picking_type_code', '!=', 'outgoing'), ('parent.state','=', 'done'), ('parent.immediate_transfer', '=', True)]}" widget="forecast_widget"/>
-                                    <field name="quantity_done" string="Done" attrs="{'readonly': [('is_quantity_done_editable', '=', False)]}"/>
+                                    <field name="quantity_done" string="Done" attrs="{'readonly': [('is_quantity_done_editable', '=', False)], 'column_invisible':[('parent.state', '=', 'draft')]}"/>
```

Comment:
I can't fill the qty_done directly in immediate transfers
![image](https://user-images.githubusercontent.com/12071695/134151003-b07bd740-ef0b-47e1-aca6-75a364fb47ac.png)

Reply:
Test should also be modify to catch it :smile:

Reply:
Nice catch, I will create our first picking Form test.